test/cpp/microbenchmarks/BUILD

```diff
@@ -73,12 +73,35 @@ grpc_cc_binary(
     deps = [":helpers"],
 )
 
+grpc_cc_library(
+    name = "fullstack_streaming_ping_pong_h",
+    testonly = 1,
+    srcs = [
+        "fullstack_streaming_ping_pong.h",
+    ],
```

Comment:
If it's a header-only library, I don't think `srcs = ` is necessary, is it? We don't use it, for example, in some targets at the top level like `gpr_codegen`

Reply:
Done